script/prepare_artifacts.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -ex
+
+mkdir -p release
+cp deployment/eks/flyte_generated.yaml ./release/flyte_eks_menifest.yaml
+cp deployment/gcp/flyte_generated.yaml ./release/flyte_gcp_menifest.yaml
+cp deployment/sandbox/flyte_generated.yaml ./release/flyte_sandbox_menifest.yaml
+cp deployment/test/flyte_generated.yaml ./release/flyte_test_menifest.yaml
```

Comment:
Can this be written in something like this:
cp deployment/{{WHATEVER}}/flyte_generated.yaml ./release/flyte_{{WHATEVER}}_menifest.yaml

just in case we end up renaming things... or if someone forks/copies this repo to create their own overlay for their company, they can continue to use these scripts..

if it's too much trouble, ignore it

Reply:
good idea